packages/core/lib/debug/interpreter.js

```diff
@@ -761,6 +765,21 @@ class DebugInterpreter {
     }
     return tempPrintouts;
   }
+
+  // parse the numbers of lines options -<num>|+<num> from user args
+  parsePrintoutLines(userArgs, currentLines) {
+    for (const argument of userArgs) {
+      if (argument.length < 2) continue;
+      const newLines = Number(argument.slice(1));
+      // ignore the arguments that are not of the correct form, number
+      if (isNaN(newLines)) continue;
+      if (argument[0] === "-") {
+        currentLines.beforeLines = newLines;
```

Comment:
As mentioned elsewhere, I think this function should be pure, and should return a value, like it was before, rather than altering its input.